src/databricks/labs/ucx/assessment/workflows.py

```diff
@@ -11,13 +11,17 @@ class Assessment(Workflow):
     def __init__(self):
         super().__init__('assessment')
 
-    @job_task(notebook="hive_metastore/tables.scala")
+    # @job_task(notebook="hive_metastore/tables.scala")
```

Comment:
let's actually remove the `tables.scala` altogether and use just the py4j wrapper, to make code more unified. make sure that `include_tables`/`include_databases` still works.